src/website/views/components/data-table/examples/portal/_complex.pug

```diff
@@ -1041,7 +1061,13 @@ script.
       table: {
         head: {
           name: { label: 'Name', sortable: true, sortBy: 'name', sortDataType: 'string', key: true, bold: true },
-          status: { label: 'Status', sortable: true, sortBy: 'status', sortDataType: 'string' },
+          status: { label: 'Status', sortable: true, sortBy: 'status', sortDataType: 'string', description: { title: 'Status', template: 'statusdesc', 
```

Comment:
The UX/UI team will probably want to remove the title on this example. Let's keep the description text only, please.